.github/workflows/build_and_release.yml

```diff
@@ -470,31 +470,44 @@ jobs:
 
 
       - name: Setup our dev environment
-        uses: ilammy/msvc-dev-cmd@v1.4.1
+        uses: ilammy/msvc-dev-cmd@v1
 
       - name: Setup ninja cmake generator
         uses: seanmiddleditch/gha-setup-ninja@master
 
       - name: Download and build our dependencies using vcpkg
-        uses: lukka/run-vcpkg@v10.2
+        uses: Lord-Kamina/vcpkg-action@master
```

Comment:
What version of vcpkg will be used here? if master then this will cause rebuilds of the cache once any of our libraries gets updated.
Preferably i'd like to specify the version we want to use so that the action always resolves to a specific version instead of latest. This way we can keep consistent builds.

Reply:
See above.

Reply:
> Also i'm missing the change to the readme where people can download the latest mingw64 build

Forgot those, will add them also.